provisioning.go

```diff
@@ -41,6 +42,8 @@ const (
 	maxTries        uint32 = 32
 	recoveryTime    uint32 = 7200
 	lockoutRecovery uint32 = 86400
+
+	srkTemplateHandle tpm2.Handle = 0x01810001
```

Comment:
does this need a comment about the choice of handle?

Reply:
I've added a comment to this now.